README.md

```diff
@@ -125,7 +108,22 @@ superwerker features can be activated individually. This enables a gradual roll-
 
 ### How do I receive updates?
 
-We plan to roll-out releases via GitHub releases. The update is then deployed via the current CloudFormation template.
+We plan to roll-out releases via GitHub releases. The update is then deployed via the current CloudFormation template. You can then perform the update according to the instructions below:
+
+1. Go to the AWS Console
+1. Navigate to the CloudFormation service
+1. Choose the superwerker stack
+1. Choose "Update"
+1. Choose "Replace current template"
+1. For "Amazon S3 URL", copy the link to the latest version of the template e.g. "https://superwerker-deployment.s3.amazonaws.com/0.12.3/templates/superwerker.template.yaml", the latest version number can be found here: [Github Releases](https://github.com/superwerker/superwerker/releases)
+1. Click "Next"
+1. Recommended: Enter an email address to "Mail address used for important notification regarding your AWS account".
```

Comment:
This instruction was special to the "M1.1", since "Mail address used for important notification regarding your AWS account", was introduced in this release. Would you think it makes sense to display it in the general update instructions?